Gruntfile.js

```diff
@@ -18,50 +19,14 @@ module.exports = function( grunt ) {
 			]
 		},
 
-		// VVV (Varying Vagrant Vagrants) Paths
-		vvv: {
-			'plugin': '/srv/www/wordpress-develop/src/wp-content/plugins/<%= pkg.name %>',
-			'coverage': '/srv/www/default/coverage/<%= pkg.name %>'
-		},
-
-		// Shell actions
-		shell: {
-			options: {
-				stdout: true,
-				stderr: true
-			},
-			readme: {
```

Comment:
These are now available through `npm run readme` and `npm test`. Not sure about the coverage report.